releases/unreleased/[phabricator]-new-blacklist-ids-argument-added.yml

```
@@ -0,0 +1,12 @@
+---
+title: '[phabricator] New blacklist-ids argument added'
+category: added
+author: Quan Zhou <quan@bitergia.com>
+issue: null
+notes: >
+    Perceval will not fetch the ids in the `--blacklist-ids` argument.
+
+    In this example `perceval` will skipt the id `123`:
+    ```
+    perceval phabricator <PHAB_URL> -t <API_TOKEN> --blacklist-ids 123
+    ```
```

```suggestion
    Perceval will not fetch the items set with the  `--blacklist-ids` argument.
    In the following example, `perceval` will skip the items with ids `123`
    and `456`:
    ```
    perceval phabricator <PHAB_URL> -t <API_TOKEN> --blacklist-ids 123 456
    ```
```